bin/adapt-varnomen-pages

```diff
@@ -0,0 +1,150 @@
+    try:
+        title = m.group(1)
+        content = re.sub(r'# \{\{\s*page.title\s*\}\}', "# " + title, content)
+    except AttributeError:
+        pass
+    
+    # simple string replacements
+    content = content.replace('{:target="\_blank"}', "")
+    content = content.replace('{:target="blank"}', "")
+    content = content.replace("{:#version-list}", "")
+    content = content.replace("/bg-material/", "/background/")
+
+    # remove header
+    content = re.sub(r"^---.+?---\s*", "", content, flags=re.S)
+    
+    # remove --- lines
+    content = re.sub(r"^---\n", "", content, flags=re.MULTILINE)
+    
+    # `### **...**`` → drop emphasis
+    content = re.sub(r"^(#+)\s+\*\*(.+?)\*\*", r"\1 \2", content, flags=re.MULTILINE)
+
+    # remove separators
+    content = re.sub(r"^\* \* \*\s*$", "", content, flags=re.MULTILINE)
+    
+    # standardize case of SVD-WG references (all uppercase), including filenames
+    content = re.sub(r"svd-wg(\d+)", r"SVD-WG\1", content)
+
+    # replace FF0000 with bold
+    content = re.sub(r'<font color="#FF0000">(.+?)</font>', r"**\1**", content)
+
+    # remove overstyling
+    content = re.sub(r"_\*\*(.+?)\*\*_", r"**\1**", content)
+    content = re.sub(r"\*\*_(.+?)_\*\*", r"**\1**", content)
+
+    # remove useless <br> on lists
+    content = re.sub(r"^(\*.+)<br>\s*$", r"\1", content, flags=re.MULTILINE)
+
+    # table reformatting    
+    content = content.replace("{:.table .table-bordered}", "")
+    content = re.sub(r"\n\s*\|(\s+\|)+\n", r"\n", content)
+    content = content.replace("|:--------------------------------------:|:-------------:|", "|:--------------------------------------|:-------------:|") 
+
+    # remove spaces around list markers, replace tabs with space, remove eol space
+    content = re.sub(r"^(\s*\*)\s+", r"\1 ", content, flags=re.MULTILINE)
+    content = re.sub(r"\t", r"    ", content, flags=re.MULTILINE)
+    content = re.sub(r"\s+$", r"", content)
+
+    # image paths refer to the baseurl, which is no longer needed   
+    content = content.replace("{{site.baseurl}}", "")
+    content = content.replace("{:.img-responsive .center-block #refSeqFig}", "")
+ 
+    # pull definitions up with preceeding line
+    content = re.sub(r"\n\s+:\s+", r": ", content)
+ 
+    # remove extraneous newlines
+    content = re.sub(r"\n\n+", r"\n\n", content)
+ 
+    # convert jekyll's {:tag} to an html anchor
+    content = re.sub(r"{:#(\w+)}", r'<a name="\1"/>', content)
+ 
+    # links to recommendations have needless /variant/ level   
+    content = content.replace("/variant/", "/")
+    
+    return content
+
+
+def process_hvnc(content: str) -> str:
+    content = re.sub(r"### (Call for members)\n\n(.+?)\n\n", r'!!! note "\1"\n\n    \2\n\n', content)
+    return content
+
+def process_recommendation(path) -> str:
+    def rewrite_discussion(d):
+        v = re.sub(r"^\s+([^:]+):\s*(.+)", r'\n!!! note "\1"\n\n    \2', d.group(1), flags=re.MULTILINE)
```

Comment:
The end of the question isn't always the first colon. The separator may be `?:`, but I didn't test if that applies to all pages. E.g., the last three comments [on this page](https://hgvs-nomenclature.org/en/17-migrate-web-site-to-markdown-and-mkdocs/recommendations/DNA/substitution/) are partially included in the body of the note. Splitting on `?:` and putting back the `?` at the end of the question would split the notes on that page correctly.